sinonjs/sinon
# Update dependencies, including nise

#### Purpose (TL;DR) - mandatory
Removes warnings from NPM (except those about `acord*` peer dependencies)

 #### How to verify - mandatory
1. `npm install sinon` (see warnings) from another dir
2. Clone this repo and enter it
2. `npm install` (no warnings)

 #### Checklist for author
